src/apiProxy/apiConfigCreators.js

```diff
@@ -71,6 +71,16 @@ export function event(params) {
 }
 
 /**
+ * admin endpoints
+ */
+export function admin(params) {
+	return {
+		endpoint,
```

Comment:
Ok, this is a little confusing. I tried to do away with the apiConfigCreator and just use the endpoint in the photosQuery, but I got an error about the type being undefined.